baggers.js

```diff
@@ -13313,6 +13313,13 @@ var data = {
       ],
       location: 'Nantes',
       sessions: [
+        {
+          title: 'Living documentation as test',
+          abstract:
+            'Maintenir la documentation sur un projet est un vrai challenge. J\explore depuis quelques temps la production d\'une Living Documentation validant l\'application et se substituant aux tests. Je propose de vous décrire le concept, soit à travers une présentation, soit en animant un coding dojo, soit en travaillant directement sur votre application.',
```

Comment:
```suggestion
            'Maintenir la documentation sur un projet est un vrai challenge. J\'explore depuis quelques temps la production d\'une Living Documentation validant l\'application et se substituant aux tests. Je propose de vous décrire le concept, soit à travers une présentation, soit en animant un coding dojo, soit en travaillant directement sur votre application.',
```

Il y a toujours le souci avec le `'`...